event-exporter/example/event-exporter.yaml

```diff
@@ -17,19 +17,18 @@ kind: ServiceAccount
 metadata:
   name: event-exporter-sa
 ---
-apiVersion: rbac.authorization.k8s.io/v1alpha1
+apiVersion: rbac.authorization.k8s.io/v1beta1
 kind: ClusterRoleBinding
 metadata:
-  name: event-exporter-rb
+  name: event-exporter-rb-default
```

Comment:
Had to create a new one, kube would not let me update the existing one.

Reply:
Maybe append a version instead? v1beta will change to v1 at some point anyway

Reply:
Or I can append `-example`?

Reply:
Sorry, I realized I don't understand the premises

> because that was being used by kube-system.

What do you mean?

Reply:
Oh, do you mean it doesn't work in a cluster with event-exporter already running as part of the deployment?

Reply:
Yes, that is what I mean, because they are cluster wide (not namespaced).

Reply:
OK, that's fine with me